doc/whats_new/v1.1.rst

```diff
@@ -1131,6 +1131,10 @@ Changelog
 :mod:`sklearn.svm`
 ..................
 
+- |Feature| :class:`svm.QuantileSVR` implements quantile regression with support vector
+  machines as derived in Hwang, C., Shim, J. (2005), DOI: 10.1007/11539087_66
+  :pr:`23153` by :user:`Alexander Trettin <atrettin>`.
```

Comment:
This will need to be moved to v1.2.rst (sorry for the slow feedback...).

Reply:
Done!